### ISSUE TITLE
isosurface data rescaling

### ISSUE THREAD

**Issue by oroszl** [maartenbreddels/ipyvolume#87]
It would be nice to have the option in the `plot_isosurface` routine such that the generated isosurface coordinates are mapped back to a user supplied box. 
This can now be done by manually scaling back  x,y,z attributes of the generated mesh:
```python
m = p3.plot_isosurface(data,level=0.05)
m.x = (m.x / 300) / (2*np.pi) - np.pi
```

**Comment by themmes** [comment]
I am not quite sure if I understand you correctly, however for the scatterplot I was able to set the axis as follows:

```
fig.ylim = [df.Y.min(), df.Y.max()]
fig.xlim = [df.X.min(), df.X.max()]
fig.zlim = [df.Z.min(), df.Z.max()]
```

Inspired by the answer here; https://stackoverflow.com/questions/2849286/python-matplotlib-subplot-how-to-set-the-axis-range

**Comment by maartenbreddels** [comment]
Solved in #117
